apps/aechannel/test/aesc_htlc_SUITE.erl

```diff
@@ -110,9 +110,10 @@ suite() ->
 init_per_suite(Config) ->
     case aec_governance:get_network_id() of
         Id when Id == <<"local_iris_testnet">>;
-                Id == <<"local_ceres_testnet">> ->
+                Id == <<"local_arcus_testnet">> ->
             aesc_fsm_SUITE:init_per_suite([{symlink, "latest.aesc_htlc"} | Config]);
         Other ->
+            %% confusing skip. This is running on both Iris and latest it seems
```

Comment:
A more accurate guess is that it runs on all protocols from Iris and onwards... The skip reason could be `only_from_iris` - skipping `ceres` makes little sense?